### ISSUE TITLE
Add "normalize games" setting like Legacy Games plugin

### ISSUE THREAD

**Issue by instinctualjealousy** [hawkeye116477/playnite-nile-plugin#15]
### Prerequisites

- [X] I've looked through [existing issues](https://github.com/hawkeye116477/playnite-nile-plugin/issues) to make sure that this feature hasn't been requested before .

### Is your feature request related to a problem? Please describe.

"Problem" is a strong word here. It does lead to metadata mismatch to not rename these games, however.
I've pushed the issue to the official Amazon integration first: JosefNemec/PlayniteExtensions#424

I think it should be replicated in this Nile plugin too. I manually fixed my games and updated metadata from scratch for now.

### Describe the solution you'd like

A menu option in the plugin's settings akin to the official Legacy Games plugin (it expands "- CE" to "Collector's Edition" on game import, you can look at the source to be sure of exactly what it's doing if there's any other rules to it):
![image](https://github.com/user-attachments/assets/6143c166-6536-41ad-bd57-4f31ef743e88)

### Describe alternatives you've considered

Not for you, but a plugin or a global Playnite setting to do just this on imported games regardless of platform.

### Additional context

Basically, many of these games are on Steam or other metadata providers with the name "Collector's Edition" instead of the "- CE" shorthand that Amazon and Legacy Games are both using for this. It's usually those low-mid budget point and click games with the somewhat cheesy art and unbelievable plots.

**Comment by hawkeye116477** [comment]
Hmm. I don't know if that checkbox would be even needed. I mean that if there is a problem with metadata, then maybe better do that without asking :smile:.

**Comment by instinctualjealousy** [comment]
I feel similarly, but I thought I'd recommend based on the official plugin's use of a setting. It's your call if you want it hardcoded, I don't know why the other would ever be desirable unless it's just for user transparency.

**Comment by instinctualjealousy** [comment]
![image](https://github.com/user-attachments/assets/0d7f3f92-9db7-4f10-8a3e-2bb2da466bc4)
I claimed ReDrawn today. Your code is keeping the hyphen- it's not supposed to. That's why in the initial issue message, I included   "- " in my quoted string. "- CE" is supposed to be replaced with "Collector's Edition", not "- Collector's Edition".

**Comment by hawkeye116477** [comment]
>  "- CE" is supposed to be replaced with "Collector's Edition", not "- Collector's Edition"

Well, you took Legacy Games plugin as example and it isn't removing hyphens. 
I checked `Mystery Case Files: Black Crown - Collector's Edition` and looks like in IGDB there is only title with hyphen, there is no such without it.
